src/templates/dandelion/components/TokenSelector/TokenSelectorInstance.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import styled from 'styled-components'
+import { addressesEqual, GU, tokenIconUrl } from '@aragon/ui'
+import { shortenAddress } from '../../../../web3-utils'
+import { ETHER_TOKEN_FAKE_ADDRESS } from '../../config/helpers/tokens'
+
+/* eslint-disable react/prop-types */
+const TokenSelectorInstance = React.memo(function TokenSelectorInstance({
+  address,
+  name,
+  symbol,
+  showIcon = true,
+}) {
+  return (
+    <div
+      css={`
+        display: flex;
+        align-items: center;
+      `}
+    >
+      {showIcon ? (
+        <Icon src={tokenIconUrl(address)} />
```

Comment:
Ok, this is my fault for suggesting `tokenIconUrl()`, but it doesn't handle "fake" testnet tokens (like the hardcoded testnet DAI).

I hope this isn't a big issue (if it is, feel free to revert to the old source). Hopeful it'll be easier to apply overrides like this in the future: https://github.com/aragon/aragon-ui/issues/717.